tcms/testcases/views.py

```diff
@@ -1696,7 +1697,7 @@ def tag(request):
                     return HttpResponse(json.dumps(ajax_response))
         return HttpResponse(json.dumps(ajax_response))
 
-    form = CaseTagForm(initial={'tag': request.REQUEST.get('o_tag')})
+    form = CaseTagForm(initial={'tag': request.POST.get('o_tag')})
```

Comment:
This is why we use git, if we do conflict one of the PRs gets precedence (I'd say the one which was opened first) and the other one simply rebases and resolves the conflict once the first PR is merged.

I have split this change into a separate commit b/c it is independent of the other changes but I'm not going to remove it.